github/github_test.go

```diff
@@ -2206,3 +2206,36 @@ func TestRateLimits_Marshal(t *testing.T) {
 
 	testJSONMarshal(t, u, want)
 }
+
+func TestParseTokenExpiration(t *testing.T) {
+	tests := []struct {
+		res    *http.Response
+		header string
+		want   Timestamp
+	}{
+		{
+			res: &http.Response{
+				Request: &http.Request{},
+				Header:  http.Header{},
+			},
+			header: "",
+			want:   Timestamp{},
+		},
+		{
+			res: &http.Response{
```

Comment:
Since `res` starts out the same in every test case, you can remove it from this struct and initialize a new fresh one below within the test loop.